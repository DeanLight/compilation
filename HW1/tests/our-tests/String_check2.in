(this is a string) (we can \( \) escape stuff here) (like \n \t \\ \142)
(now we are\
on a new line)
(hel\
lo) (is it me) (youre\
\
looking for)

<3838> <2A2A>
<30
31><38
39>
<2 b 2 B      2 B>
<2            
F>

<><><>()()
()