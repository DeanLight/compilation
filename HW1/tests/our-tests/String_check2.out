1 STRING this is a string
1 STRING we can ( ) escape stuff here
1 STRING like 
 	 \ b
3 STRING now we areon a new line
5 STRING hello
5 STRING is it me
7 STRING yourelooking for
9 STRING 88
9 STRING **
11 STRING 01
12 STRING 89
13 STRING +++
15 STRING /
17 STRING 
17 STRING 
17 STRING 
17 STRING 
17 STRING 
18 STRING 
18 EOF 
